safety/scan/command.py

```diff
@@ -250,7 +356,11 @@ def scan(ctx: typer.Context,
                             typer.Option("--apply-fixes",
                                 help=SCAN_APPLY_FIXES,
                                 show_default=False)
-                        ] = False
+                        ] = False,
+        filter_keys: Annotated[
```

Comment:
nit: Can you format this lie and line #578, the line wraps appear to be missing.

Reply:
I tried to fix this formatting, let me know if it got what we wanted